service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/ObservationMapper.java

```diff
@@ -278,6 +288,34 @@ private CommentType prepareCommentType(Observation observation) {
             ? CommentType.USER_COMMENT : CommentType.AGGREGATE_COMMENT_SET;
     }
 
+    private String prepareReferenceRangeValues(SimpleQuantity simpleQuantity) {
+        return Stream.of(
```

Comment:
I think this can be:
```
        return Stream.of(
            Optional.ofNullable(simpleQuantity.getValue()),
            Optional.ofNullable(simpleQuantity.getUnit())
            )
            .flatMap(Optional::stream)
            .filter(Objects::nonNull)
            .map(Object::toString)
            .collect(Collectors.joining(StringUtils.SPACE));
```